docs/reference/autoscaling/apis/get-autoscaling-policy.asciidoc

```diff
@@ -0,0 +1,67 @@
+[role="xpack"]
+[testenv="platinum"]
+[[autoscaling-get-autoscaling-policy]]
+=== Get autoscaling policy API
+++++
+<titleabbrev>Get autoscaling policy</titleabbrev>
+++++
+
+Get autoscaling policy.
+
+[[autoscaling-get-autoscaling-policy-request]]
+==== {api-request-title}
+
+[source,console]
+--------------------------------------------------
+PUT /_autoscaling/policy/my_autoscaling_policy
+{
+  "policy": {
+    "deciders": {
+      "always": {
+      }
+    }
+  }
+}
+--------------------------------------------------
+// TESTSETUP
+
+[source,console]
+--------------------------------------------------
+GET /_autoscaling/policy/<name>
+--------------------------------------------------
+// TEST[s/<name>/my_autoscaling_policy/]
+
+[[autoscaling-get-autoscaling-policy-prereqs]]
+==== {api-prereq-title}
+
+* If the {es} {security-features} are enabled, you must have
+`manage_autoscaling` cluster privileges. For more information, see
+<<security-privileges>>.
+
+[[autoscaling-get-autoscaling-policy-desc]]
+==== {api-description-title}
+
+This API gets an autoscaling policy with the provided name.
+
+[[autoscaling-get-autoscaling-policy-examples]]
+==== {api-examples-title}
+
+This example gets an autoscaling policy named `my_autosaling_policy`.
+
+[source,console]
+--------------------------------------------------
+GET /_autoscaling/policy/my_autoscaling_policy
+--------------------------------------------------
+// TEST
+
+The API returns the following result:
+
+[source,console-result]
+--------------------------------------------------
+{
```

Comment:
I wonder if we need an array to prepare for listing all deciders? I guess we can state we will never add wildcard support and hence it is two different endpoints, i.e., this is OK?

Reply:
Yeah, I do not think that we should support wildcards, or getting multiple policies at once (comma-separated). In general, I think that it greatly complicates our APIs (e.g., what do we do if one policy exists, and another doesn't? we have faced these challenges with existing APIs and I think it's just complicated our APIs and their implementations with little benefit to our users).